tsconfig.json

```diff
@@ -1,17 +1,22 @@
 {
   "compilerOptions": {
-    "allowJs": true,
-    "baseUrl": ".",
+    "allowJs": false,
```

Comment:
Not sure why this was ever set to be honest. I suspect copy/paste of tsconfig from somewhere without really knowing what is happening. We don't need to be touching any js.